### ISSUE TITLE
UHP: Fix content width & add outline border

### ISSUE THREAD

**Issue by kyhryt** [USAJOBS/Help#89]
@humancompanion I have two requests. 

1. Is it possible to reduce the width of the content to be two thirds: https://standards.usa.gov/grids/. The line length is too wide. This makes it difficult for readers to focus on the text. 

2. Add a border around the shell, like we did in dashboard. This is not ideal, but the content is floating in space. 

Thanks,
Ky.

**Comment by humancompanion** [comment]
@kyhryt - Let's look at this together next week. After adjusting the content in the right column I wonder if we should also shorten the width of the left column. 

As for the border, please don't make me do it. I don't feel the content is floating in space. I mean, we're all floating in space, technically, but I fixed up the left edge and I think that helps. Have a look and see what you think.

**Comment by kyhryt** [comment]
@humancompanion In the design, the left content is 3 columns. What is it in dev? 

I'm okay with not adding a border. After looking at this again, I don't think we need it. 

Ky.

**Comment by humancompanion** [comment]
@kyhryt - Your comp does have 3 columns for the left column. But it also has additional gutter padding (the gutter is wider in the comp). I've added that extra padding at the 992px breakpoint.

However, the 6 column main content column results in only 455px wide (at the narrowest). I think we're over compensating for the $text-max-width. Here is what the standards have now for their defaults:

```
// Magic Numbers
$text-max-width:      53rem !default;
$lead-max-width:      77rem !default;
$article-max-width:   600px !default;
```

I'm inclined to treat these as articles. We had overridden $text-max-width to 620px ourselves for app guide and I'd like to do the same throughout Help Center. Are you okay with a 600px max-width main content column?

**Comment by humancompanion** [comment]
@kyhryt - I've made these changes and deployed them. Please review and close this out if you are satisfied. Thanks.

**Comment by kyhryt** [comment]
close #89

**Comment by kyhryt** [comment]
closed #89

**Comment by kyhryt** [comment]
@humancompanion Looks good. For some reason, I can't close this issue.
